ztp/ztp-policy-generator/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: build generate-policy generate-crs
+build:
+	./hack/build.sh
+
+generate-policy:
```

Comment:
I don't think using the command to generate policies or crs is the right way, the PGT deiced if it will generate policy or just CR